defaults/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
 grafana_version: latest
 
+# Should the package install be forced ? If not, the package downgrading is not possible.
+grafana_force_install: no
```

Comment:
it should be `false`, not `no`.

https://travis-ci.org/cloudalchemy/ansible-grafana/jobs/465735951#L522-L524